takeshape/takeshape-starter-ecommerce-kitchen-sink
# Use reviews.io provider

> If you don't want to invoke CI, put `[skip ci]` in the commit message

### Screenshots

### Test Plan (Steps to test):

1. Connect reviews.io
2. Run a query that uses REVIEWS.io
```
{
  getProductReviews(sku:"prod_LPvIyYzsGmcQJp"){
    reviews{
      total
    }
    store{
      name      
    }
  }
}
```
3. Ensure data returned is not null.

### Checklist:
